### PR TITLE
KTOR-2330

### PR DESCRIPTION
**Pull request by andreyaksenov**
Added a tip on how to reaload Maven changes in Idea

### CODE REVIEW

File: topics/Maven.xml [hunk 1]
```diff
@@ -314,6 +314,11 @@
     </dependencies>
 </project>
                 ]]></code>
+                <tip>
+                    <p>
+                        To load changes made in <path>pom.xml</path>, you can click <control>Load Maven Changes</control> at the right part of the editor.
```
Comment (hhariri) [inline topics/Maven.xml:319]: To load changes made -> To have the changes made take effect, you can click the ....., on the Maven Tool Window.

**Review by hhariri — CHANGES_REQUESTED** [review]
Can't figure out how to actually comment on the parts not changed :(

File: topics/Maven.xml (hunk 1 above)
Comment (Stexxe) [reply topics/Maven.xml:319]: It's better to describe what the effect is.

Comment (andreyaksenov) [reply topics/Maven.xml]: rephrased a bit and added a link to IDEA docs

**Review by hhariri — APPROVED** [review]

**Pull request merged**